src/libs/SidebarUtils.ts

```diff
@@ -62,6 +62,16 @@ function compareStringDates(a: string, b: string): 0 | 1 | -1 {
     return 0;
 }
 
+/**
+ * A mini report object that contains only the necessary information to sort reports.
+ * This is used to avoid copying the entire report object and only the necessary information.
+ */
+type MiniReport = {
```

Comment:
I think we could have called this as MinimalSidebarReport or something like that to indicate this is specifically the minimum we need for the Sidebar, but NAB